entrypoints/dev.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
Could you add a blank line before and after this?  I find conceptually breaking up shell scripts into shebang, config, code really helps me read them.

Reply:
After, sure, but convention for `set ...` is definitely the next line after the shebang?

Reply:
I've always seen it with spaces either side!  But happy to go with only one after 🙂